### PR TITLE
Add hello money to qr brands

### PR DESCRIPTION
**Pull request by phieronymus**
Node part for https://univapay-systems.slack.com/archives/CA0U3CSQ1/p1681096439417129.

We will need the logo to add it to the common.

### CODE REVIEW

File: src/resources/common/enums.ts [hunk 1]
```diff
@@ -110,6 +110,7 @@ export enum QRGateway {
     ALIPAY_PLUS = "alipay_plus",
     AU_PAY = "au_pay",
     BARTONG = "bartong",
+    HELLOMONEY = "hellomoney",
```
Comment (gdp88) [inline src/resources/common/enums.ts:113]: should this be
```
HELLO_MONEY = "hello_money",
```
?

Comment (phieronymus) [reply src/resources/common/enums.ts:113]: No. As per the link on the slack thread: https://github.com/univapaycast/gyron-payments-api/pull/6000, this is `hellomoney`. Note: this is an API key, we can not change it on frontend.

Comment (gdp88) [reply src/resources/common/enums.ts:113]: oh I see thanx! resolving!

**Review by gdp88 — APPROVED** [review]

**Comment by gdp88** [comment]
lgtm! merging!

**Pull request merged**